configuration.k

```diff
@@ -0,0 +1,220 @@
+                    <difficulty>       0          </difficulty>       // H_d
+                    <number>           0          </number>           // H_i
+                    <gasLimit>         0          </gasLimit>         // H_l
+                    <timestamp>        0          </timestamp>        // H_s
+                </block>
+            </execEngine>
+            
+            // This is the part of the "Solidity VM" where we keep the available contracts. 
+            // Those are either obtained from source file (creation) or loaded from accounts (function call).
+
+            // When we parse Solidity source file we 
+            //   - desugar/preprocess modifiers, inheritance (?) etc. 
+            //   - create one <contract> structure for each contract declared in the sources
+            // (let's not worry about those for now, and just stick to simple functions)
+
+            // From here, <contract>s can be either deployed or executed. 
+            // When a contract is deployed we not only copy its <contract> cell into the <account>, 
+            // but we also copy the other contracts that were present in the same source file, 
+            // so that the deployed contract have access to them and can create and deploy them if needed. 
+            // (see the <account> cell below for more notes about this)
+
+            <contracts>
+                <Contract multiplicity = "*">
+                    <cname> .K </cname>
+                    
+                    <functions>
+                        <Function multiplicity="*">
+                            <fname>         .K         </fname>
+                            <inputParams>  .K         </inputParams> 
+                            <Returns>      .K         </Returns>
+                            <quantifiers>  .K         </quantifiers>
+                            <visibility>   .K         </visibility> // Visibility ::= public | privat
+                            <body> .K </body>
+                        </Function>
+                    </functions>
+		            
+                    <stateVar> .List </stateVar>
+                    <Constructor> false </Constructor>
+                    // ... more to be added later ...
+                
+                </Contract>
+            </contracts>
+
+        </solidity>
+
+        // Ethereum Network
+        // ==========================
+
+        <network>
+
+            // Accounts Record
+            // ---------------
+
+            <activeAccounts> .Set </activeAccounts>
+	        <cntAccount> 0:Int </cntAccount>
+            <accounts>
+                <account multiplicity="*">
+                    
+                    // Same as in KEVM / IELE
+                    
+                    <acctID>  .K                      </acctID>
+                    <balance> 0                      </balance>
+		            <context> .Map		     </context>
```

Comment:
what is this?

Reply:
The `context` cell refers to the environment, mapping variable names to logical storage references. Actually, `context` is equivalent to the `env` cell in the `execEngine`. I will change `context` to `env` to be consistent.

Reply:
I would like to change the name to `env` to be consistent with `env` in `execEngine`, but failed. It seems that K-framework complains about having the same name. Could someone help to check? Thanks.

Reply:
@shangweilin you can't have two cells named the same thing, though it would be nice.

You could go for `acctEnv` for the ones stored in the `<account>` cell?

Reply:
I have changed the cell `<context>` into `<acctEnv>`.